### PR TITLE
Adding bundle size checking

### PR DESCRIPTION
**Pull request by porcellus**
## Summary of change

Adding bundle size checking into PRs and as a script

## Related issues

-   supertokens/supertokens-core#178

## Test Plan

Build change only

## Documentation changes

(If relevant, please create a PR in our [docs repo](https://github.com/supertokens/docs), or create a checklist here highlighting the necessary changes)

## Checklist for important updates

-   [x] Changelog has been updated
-   [x] `frontendDriverInterfaceSupported.json` file has been updated (if needed)
    -   Along with the associated array in `lib/ts/version.ts`
-   [ ] Changes to the version if needed
    -   In `package.json`
    -   In `package-lock.json`
    -   In `lib/ts/version.ts`
-   [x] Had run `npm run build-pretty`
-   [x] Had installed and ran the pre-commit hook
-   [x] Issue this PR against the latest non released version branch.
    -   To know which one it is, run find the latest released tag (`git tag`) in the format `vX.Y.Z`, and then find the latest branch (`git branch --all`) whose `X.Y` is greater than the latest released tag.
    -   If no such branch exists, then create one from the latest released branch.

### CODE REVIEW

**Comment by porcellus** [comment]
I didn't update the version number, since the PR doesn't touch the code we release

**Comment by rishabhpoddar** [comment]
- This is good. However I also prefer if there is check for this in the CICD pipeline as well - when we add a dev tag. And if it crosses a certain limit, then it should fail the CICD.
- Also, if we add a new recipe, will there need to be a change to the script to add that recipe's bundle also?

**Comment by rishabhpoddar** [comment]
@porcellus Also, if we add a new recipe, will there need to be a change to the script to add that recipe's bundle also?

File: package.json [hunk 1]
```diff
@@ -79,8 +81,31 @@
...
+            "path": "recipe/thirdpartyemailpassword/index.js",
+            "limit": "110kb"
+        },
+        {
+            "path": "recipe/thirdparty/index.js",
+            "limit": "110kb"
+        },
+        {
+            "path": "recipe/emailpassword/index.js",
+            "limit": "110kb"
```
Comment (rishabhpoddar) [inline package.json:106]: Shouldn't we have lower limits on these compared to that of thirdpartyemailpassword?

Comment (rishabhpoddar) [reply package.json:106]: Also, will we have to add to this everytime we add a new recipe? If yes, you should edit the PR template's checklist for this repo to say to modify this (and any other script) if we add a new recipe.

**Review by rishabhpoddar — CHANGES_REQUESTED** [review]

**Pull request merged**